src/hot_stuff.py

```diff
@@ -0,0 +1,70 @@
+"""
+Module for analyzing data in hot_stuff.csv file
+"""
+import os
+import pandas as pd
+
+
+class HotStuff:
+    """
+    Class for storing and analyzing data in hot_stuff.csv file
+    """
+    def __init__(self):
+        self.__filepath = os.path.join(os.path.dirname(__file__), '../data', 'hot_stuff.csv')
+        self.__data = pd.read_csv(self.__filepath)
+
+    def weeks_of_number_one_hit(self, song):
+        """
+        Returns the number of weeks while the given song was hit No.1
+        """
+        return self.__data.loc[(self.__data['Song'] == song) &
+                               (self.__data['Week Position'] == 1), 'Week Position'].count()
+
+    def songs_hit_podium_but_not_no1(self):
```

Comment:
This solution is not good because it only selects records that were 2 or 3 in the first week and never 1 after that.
For example, if on the first week, the peak position was 23 then always 2, then it would not be selected, but it should be.

Reply:
I changed filter to: filter(lambda data: ((data['Peak Position'] == 2).any() | (data['Peak Position'] == 3).any()) 
& (data['Peak Position'] != 1).all()) and then count the number of unique SongID-s.

Reply:
Okay, the solution is now good.